docs/sphinx/source/index.rst

```diff
@@ -107,22 +129,3 @@ pvlib python is a `NumFOCUS Affiliated Project <https://numfocus.org/sponsored-p
 .. image:: https://i0.wp.com/numfocus.org/wp-content/uploads/2019/06/AffiliatedProject.png
   :target: https://numfocus.org/sponsored-projects/affiliated-projects
   :alt: NumFocus Affliated Projects
-
-Contents
-========
-
-.. toctree::
-   :maxdepth: 1
```

Comment:
This toctree is what defines the whole documentation website structure, so we better not delete it :P

You can add `:hidden:` if you don't want it to be displayed in the page content.